CMakeLists.txt

```diff
@@ -32,7 +32,7 @@ option(RDKSHELL_BUILD_FORCE_ANIMATE "RDKSHELL_BUILD_FORCE_ANIMATE" OFF)
 
 set(COMMUNICATIONDIR ${CMAKE_CURRENT_SOURCE_DIR}/communication)
 
-set(RDKSHELL_SOURCES rdkshell.cpp compositorcontroller.cpp essosinstance.cpp rdkcompositor.cpp linuxkeys.cpp eastereggs.cpp animation.cpp animationutilities.cpp  rdkshelldata.cpp rdkshelljson.cpp linuxinput.cpp logger.cpp)
+set(RDKSHELL_SOURCES rdkshell.cpp compositorcontroller.cpp essosinstance.cpp rdkcompositor.cpp rdkcompositor_surface.cpp rdkcompositor_nested.cpp linuxkeys.cpp eastereggs.cpp animation.cpp animationutilities.cpp  rdkshelldata.cpp rdkshelljson.cpp linuxinput.cpp logger.cpp)
```

Comment:
Please remove `_` from file names.  Please rename to `rdkcompositorsurface.cpp` and `rdkcompositor_nested.cpp`